### PR TITLE
add command arg:signatureType & finalApkPath

### PR DESCRIPTION
**Pull request by huangchuanal**

### CODE REVIEW

File: AndResGuard-cli/build.gradle [hunk 1]
```diff
@@ -37,4 +39,8 @@ task buildJar(type: Copy, dependsOn: [build, jar]) {
     into('../tool_output')
 }
 
+def releaseTime() {
+    return new Date().format("yyyy-MM-dd HH:mm ZZZ", TimeZone.getTimeZone("GMT+0800"))
```
Comment (simpleton) [inline AndResGuard-cli/build.gradle]: Don't hardcode the timezone.

Comment (simpleton) [inline AndResGuard-cli/build.gradle]: Don't hardcode the timezone.

File: AndResGuard-cli/src/main/java/com/tencent/mm/resourceproguard/cli/CliMain.java [hunk 2]
```diff
@@ -352,6 +383,23 @@ public ReadArgs invoke() {
                     }
                     storealias = args[++index];
                     mSetSignThroughCmd = true;
+                } else if (arg.equals(ARG_SIGNATURE_TYPE)) {
+                    if (index == args.length - 1) {
+                        System.err.println("Missing signature type argument");
+                        goToError();
+                    }
+                    boolean v2Enabled = false;
```
Comment (simpleton) [inline AndResGuard-cli/src/main/java/com/tencent/mm/resourceproguard/cli/CliMain.java]: This temp variable seems unnecessary.

File: AndResGuard-core/src/main/java/com/tencent/mm/androlib/ResourceApkBuilder.java [hunk 3]
```diff
@@ -65,12 +65,12 @@ public void buildApkWithV1sign(HashMap<String, Integer> compressData) throws IOE
     }
 
     private void copyFinalApkV1() throws IOException {
-        if (StringUtil.isPresent(finalApkBackupPath)) {
-            System.out.println(String.format("Backup Final APk(V1) to %s", finalApkBackupPath));
+        if (null != finalApkFile) {
```
Comment (simpleton) [inline AndResGuard-core/src/main/java/com/tencent/mm/androlib/ResourceApkBuilder.java]: I think obj !=null is generally more readable.

File: AndResGuard-core/src/main/java/com/tencent/mm/androlib/ResourceApkBuilder.java [hunk 4]
```diff
@@ -94,9 +94,9 @@ public void buildApkWithV2sign(HashMap<String, Integer> compressData) throws Exc
     }
 
     private void copyFinalApkV2() throws IOException {
-        if (mSignedApk.exists() && StringUtil.isPresent(finalApkBackupPath)) {
-            System.out.println(String.format("Backup Final APk(V2) to %s", finalApkBackupPath));
-            FileOperation.copyFileUsingStream(mSignedApk, new File(finalApkBackupPath));
+        if (mSignedApk.exists() && null != finalApkFile) {
```
Comment (simpleton) [inline AndResGuard-core/src/main/java/com/tencent/mm/androlib/ResourceApkBuilder.java]: obj !=null

Comment (huangchuanal) [reply AndResGuard-core/src/main/java/com/tencent/mm/androlib/ResourceApkBuilder.java]: my bad habit from C. Thanks.

Comment (simpleton) [reply AndResGuard-core/src/main/java/com/tencent/mm/androlib/ResourceApkBuilder.java]: That's not bad, just make less sense.

**Pull request closed without merging**